### PR TITLE
feat: ui change in api tokens

### PR DESCRIPTION
**Pull request by rish2320**
# Description

Change in the appearance of the API Tokens page.

Fixes # [AB2383](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=2383)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
1: To start with , I implemented the changes in the files and designed the front-end as required .
2: Then i verified the design on the localhost and saw if all the features were working fine and they did.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/assets/icons/ic-help-filled.svg
Comment (milindk8) [inline src/assets/icons/ic-help-filled.svg:1]: This icon is already available use that one & remove this one

File: src/components/apiTokens/APITokenList.tsx [hunk 1]
```diff
@@ -11,6 +11,15 @@ import DeleteAPITokenModal from './DeleteAPITokenModal'
 import NoResults from '../../assets/img/empty-noresult@2x.png'
 import './apiToken.scss'
 import EmptyState from '../EmptyState/EmptyState'
+import { ReactComponent as Question } from '../../assets/icons/ic-help-outline.svg'
+import { ReactComponent as QuestionFilled } from '../../assets/icons/ic-help-filled.svg'
+import TippyCustomized from '../common/TippyCustomized'
+
+
+enum TippyTheme {
```
Comment (milindk8) [inline src/components/apiTokens/APITokenList.tsx:19]: We have classes for tippy for both black & white. Verify that the theme & classes have same attributes

File: src/components/apiTokens/APITokenList.tsx [hunk 2]
```diff
@@ -39,26 +48,55 @@ function APITokenList({ tokenList, renderSearchToken, reload }: APITokenListType
...
     return (
-        <div>
-            <div className="cn-9 fw-6 fs-16">API tokens</div>
-            <p className="fs-12 fw-4">Tokens you have generated that can be used to access the Devtron API.</p>
-            <div className="flex dc__content-space mb-16">
-                <button className="flex cta h-32" onClick={() => handleGenerateRowActionButton('create')}>
-                    Generate new token
-                </button>
-                {renderSearchToken()}
+        <div className='white-background'>
```
Comment (milindk8) [inline src/components/apiTokens/APITokenList.tsx:77]: Can't we use bcn-0 instead

File: src/components/apiTokens/APITokenList.tsx [hunk 3]
```diff
@@ -39,26 +48,55 @@ function APITokenList({ tokenList, renderSearchToken, reload }: APITokenListType
...
     }
+
+    const handleQuestion = () => {
+        return (
+            <TippyCustomized
+                theme={TippyTheme.white}
+                className="w-300"
+                placement="right"
+                Icon={QuestionFilled}
+                heading={'API tokens'}
```
[the remark this answers is not in the document]
Comment (rish2320) [reply src/components/apiTokens/APITokenList.tsx:59]: done

File: src/components/apiTokens/APITokenList.tsx (hunk 1 above)
Comment (rish2320) [reply src/components/apiTokens/APITokenList.tsx:19]: done

File: src/components/apiTokens/APITokenList.tsx (hunk 2 above)
Comment (rish2320) [reply src/components/apiTokens/APITokenList.tsx:77]: done

File: src/components/apiTokens/__tests__/ApiTokens.test.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import '@testing-library/jest-dom'
+import { fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import APITokenList from '../APITokenList';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('APITokenList', () => {
+
+    const tokenList   = [
```
[the remark this answers is not in the document]
Comment (rish2320) [reply src/components/apiTokens/__tests__/ApiTokens.test.tsx:10]: done

File: src/components/apiTokens/APITokenList.tsx [hunk 5]
```diff
@@ -39,26 +48,55 @@ function APITokenList({ tokenList, renderSearchToken, reload }: APITokenListType
...
-                {renderSearchToken()}
+        <div className='white-background'>
+            <div className='flex dc__content-space pl-20 pr-20'>
+                <div className='flex row ml-0'>
+                    <div className="cn-9 fw-6 fs-16">API tokens</div>
+                    {handleQuestion()}
+                </div>
+                <div className="flex dc__align-end dc__content-end mb-8 mt-16">
+                    {renderSearchToken()}
+                    <button className="flex cta h-32 ml-10 app-status-card__divider" onClick={() => handleGenerateRowActionButton('create')}>
```
[the remark this answers is not in the document]
Comment (rish2320) [reply src/components/apiTokens/APITokenList.tsx:85]: done

File: src/components/apiTokens/APITokenList.tsx [hunk 6]
```diff
@@ -39,26 +48,55 @@ function APITokenList({ tokenList, renderSearchToken, reload }: APITokenListType
...
+                <div className="api-list-row fw-6 cn-7 fs-12 dc__border-bottom pt-8 pb-8 pr-20 pl-20 dc__uppercase">
                     <div></div>
                     <div>Name</div>
                     <div>Last Used On</div>
                     <div>Last used by Ip add.</div>
                     <div>Expires on</div>
                     <div></div>
                 </div>
-                <div className="dc__overflow-scroll" style={{'height': 'calc(100vh - 153px)'}}>
+                <div className="dc__overflow-scroll" style={{'height': 'calc(100vh - 141px)'}}>
```
[the remark this answers is not in the document]
Comment (rish2320) [reply src/components/apiTokens/APITokenList.tsx:99]: done

File: src/assets/icons/ic-help-filled.svg
Comment (rish2320) [reply src/assets/icons/ic-help-filled.svg:1]: done

**Review by milindk8 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**